recipes/h5pp/all/conanfile.py

```diff
@@ -1,6 +1,9 @@
+from conan import ConanFile
 from conan.tools.microsoft import is_msvc
-from conans import ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conan.tools.files import get
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
 import os
 
 required_conan_version = ">=1.45.0"
```

Comment:
[check_min_cppstd](https://docs.conan.io/en/latest/reference/conanfile/tools/build.html#conan-tools-build-check-min-cppstd) is only available since 1.50.0. Can you update the minimum Conan version required to reflect this?

```suggestion
required_conan_version = ">=1.50.0"
```